protoc_plugin/lib/message_generator.dart

```diff
@@ -523,6 +523,19 @@ class MessageGenerator extends ProtobufContainer {
                 fieldPathSegment: memberFieldPath,
                 start: 'void '.length)
           ]);
+      if (field.baseType.isMessage) {
```

Comment:
`&& !field.isRepeated`

?

Reply:
Not necessary, since we are already in the else branch of isRepeated.